src/main/resources/static/js/hposummary.js

```diff
@@ -1,6 +1,11 @@
 // Javascript for the hposummary table on phenotypes view
 
 $(document).ready(function() {
+	
+	function getContextPath() {
+		return window.location.pathname.substring(0, window.location.pathname.indexOf("/",2));
```

Comment:
This looks okay. If the second `/` is ever missing, you could also do something like this:
```
'/' + window.location.pathname.split('/')[1]
```

Reply:
That does look a little nicer. I already merged but will go back and fix in a subsequent branch